src/Benqi/config/avalanche.json

```diff
@@ -0,0 +1,64 @@
+{
+    "network": "avalanche",
+    "adapter": "Benqi",
+    "contracts": [
+      {
+        "contract": "Token",
+        "abis": ["Comptroller","Token"],
+        "entities": ["BenqiTokenData"],
+        "events": [
+          {
+            "event-name": "AccrueInterest",
+            "event-params": "(uint256,uint256,uint256,uint256)"
+          },
+          {
+            "event-name": "Borrow",
+            "event-params": "(address,uint256,uint256,uint256)"
+          },
+          {
+            "event-name": "LiquidateBorrow",
+            "event-params": "(address,address,uint256,address,uint256)"
+          },
+          {
+            "event-name": "Mint",
+            "event-params": "(address,uint256,uint256)"
+          },
+          {
+            "event-name": "NewReserveFactor",
+            "event-params": "(uint256,uint256)"
+          },
+          {
+            "event-name": "Redeem",
+            "event-params": "(address,uint256,uint256)"
+          },
+          {
+            "event-name": "RepayBorrow",
+            "event-params": "(address,address,uint256,uint256,uint256)"
+          },
+          {
+            "event-name": "ReservesAdded",
+            "event-params": "(address,uint256,uint256)"
+          },
+          {
+            "event-name": "ReservesReduced",
+            "event-params": "(address,uint256,uint256)"
+          },
+          {
+            "event-name": "Transfer",
+            "event-params": "(indexed address,indexed address,uint256)"
+          }
+        ],
+        "instances": [
+          { "address": "0x5C0401e81Bc07Ca70fAD469b451682c0d747Ef1c", "startBlock": 7710735, "symbol": "qiAVAX" },
+          { "address": "0xe194c4c5aC32a3C9ffDb358d9Bfd523a0B6d1568", "startBlock": 10960099, "symbol": "qiBTC" },
+          { "address": "0x334AD834Cd4481BB02d09615E7c11a00579A7909", "startBlock": 8983575, "symbol": "qiETH" },
+          { "address": "0xc9e5999b8e75C3fEB117F6f73E664b9f3C8ca65C", "startBlock": 7710758, "symbol": "qiUSDT" },
+          { "address": "0x4e9f683A27a6BdAD3FC2764003759277e93696e6", "startBlock": 7710755, "symbol": "qiLINK" },
+          { "address": "0xBEb5d47A3f720Ec0a390d04b4d41ED7d9688bC7F", "startBlock": 10921410, "symbol": "qiUSDC" },
+          { "address": "0x835866d37AFB8CB8F8334dCCdaf66cf01832Ff5D", "startBlock": 7710760, "symbol": "qiDAI" },
```

Comment:
How did you come up with these `startBlock` values? I checked one of them and it is a totally different number.